custom-recipes/sharepoint-online-append-list/recipe.json

```diff
@@ -0,0 +1,185 @@
+            ]
+        },
+        {
+            "name": "sharepoint_oauth",
+            "label": "Azure preset",
+            "type": "PRESET",
+            "parameterSetId": "oauth-login",
+            "visibilityCondition": "model.auth_type == 'oauth'"
+        },
+        {
+            "name": "sharepoint_sharepy",
+            "label": "SharePoint preset",
+            "type": "PRESET",
+            "parameterSetId": "sharepoint-login",
+            "visibilityCondition": "model.auth_type == 'login'"
+        },
+        {
+            "name": "site_app_permissions",
+            "label": "Site App preset",
+            "type": "PRESET",
+            "parameterSetId": "site-app-permissions",
+            "visibilityCondition": "model.auth_type == 'site-app-permissions'"
+        },
+        {
+            "name": "sharepoint_list_title",
+            "label": "List title",
+            "defaultValue":  "DSS_${projectKey}_",
+            "description": "",
+            "type": "STRING",
+            "mandatory": true
+        },
+        {
+            "name": "expand_lookup",
+            "label": "Expand lookup fields",
+            "description": "",
+            "type": "BOOLEAN",
+            "defaultValue": false,
+            "visibilityCondition": false,
+            "mandatory": true
+        },
+        {
+            "name": "metadata_to_retrieve",
+            "label": "Metadata to retrieve",
+            "type": "MULTISELECT",
+            "visibilityCondition": false,
+            "selectChoices": [
+                {
+                    "value": "ID",
+                    "label": "Item's ID"
+                },
+                {
+                    "value": "Created",
+                    "label": "Creation date"
+                },
+                {
+                    "value": "Modified",
+                    "label": "Modification date"
+                },
+                {
+                    "value": "Author",
+                    "label": "Created by"
+                },
+                {
+                    "value": "Editor",
+                    "label": "Modified by"
+                }
+            ]
+        },
+        {
+            "name": "advanced_parameters",
+            "label": "Show advanced parameters",
+            "description": "",
+            "type": "BOOLEAN",
+            "defaultValue": false
+        },
+        {
+            "name": "sharepoint_site_overwrite",
+            "label": "Site path preset overwrite",
+            "type": "STRING",
+            "description": "sites/site_name/subsite...",
+            "visibilityCondition": "model.advanced_parameters == true"
+        },
+        {
+            "name": "sharepoint_list_view_title",
+            "label": "View name",
+            "description": "Read data from a specific view",
+            "type": "STRING",
+            "defaultValue": "",
+            "visibilityCondition": "model.advanced_parameters == true"
```

Comment:
Do we need this param? If we are writing data does this do anything?

Reply:
👍 removed in [c4fabcf](https://github.com/dataiku/dss-plugin-sharepoint-online/pull/47/commits/c4fabcf3486c0379c3dec03cd8a2f093d0e04075)